services/QuillLMS/spec/services/analytics/segment_analytics_spec.rb

```diff
@@ -72,9 +72,10 @@
     let(:teacher) { create(:teacher) }
     let(:activity) { create(:diagnostic_activity) }
     let(:student) { create(:student) }
+    let(:session) { create(:activity_session) }
```

Comment:
For consistency, I would name this `activity_session` and also `session` generally means something else.